FlowCrypt/Resources/flowcrypt-ios-prod.js.txt

```diff
@@ -20636,34 +20636,34 @@ var time_estimates;time_estimates={estimate_attack_times:function(e){var t,n,s,o
   /******/ (() => { // webpackBootstrap
```

Comment:
On other codebases (though not yet on Android) we have a habit of placing generated code into a `../generated/..` folder. I think for this file that would be appropriate. Let's file that for another issue. (the alternative would be to name it `generated-flowcrypt-ios-prod.js.txt`)

Reply:
https://github.com/FlowCrypt/flowcrypt-ios/issues/1854